chromium/pages/popup/style.css

```diff
@@ -76,7 +76,6 @@ footer a.button {
 
 /* Elements */
 .settings_block {
```

Comment:
Seeing a display issue with this branch where the grey horizontal line is now crossing the "see more" text with English. Going to see if this can be amended for both by aligning `display: block` with padding.
![image](https://user-images.githubusercontent.com/4276080/66341751-c66dd480-e8fc-11e9-93aa-b1f08241c49b.png)

Reply:
I added additional `padding` to to make the "See more" element align better in ea8ee20